parser/SqlLexer.lpp

```diff
@@ -120,6 +120,7 @@ unsigned_numeric_literal {exact_numeric_literal}|{approximate_numeric_literal}
 "between"          return TOKEN_BETWEEN;
 "bigint"           return TOKEN_BIGINT;
 "bit"              return TOKEN_BIT;
+"bloom_filter"     return TOKEN_BLOOM_FILTER_INDEX;
```

Comment:
SQL keywords usually don't have underscores, so maybe call this "bloomfilter" instead.